### PR TITLE
Added flag to skip GlobalNet configuration on join.

### PR DESCRIPTION
**Pull request by renan-campos**
Signed-off-by: Renan Campos <rcampos@redhat.com>

### CODE REVIEW

**Review by skitt — CHANGES_REQUESTED** [review]
Thanks for this @renan-campos! We’re trying to make boolean flags consistently “positive”, _i.e._ setting them enables something (see https://github.com/submariner-io/submariner-operator/issues/649); could you please rework this as

    --allocate-globalnet

instead, with a default value of `true`? Users would then use

    --allocate-globalnet=false

to skip allocation.

**Review by skitt — APPROVED** [review]
Thanks!

File: pkg/subctl/cmd/join.go [hunk 1]
```diff
@@ -89,6 +90,7 @@ func addJoinFlags(cmd *cobra.Command) {
 	cmd.Flags().IntVar(&nattPort, "nattport", 4500, "IPsec NATT port")
 	cmd.Flags().IntVar(&ikePort, "ikeport", 500, "IPsec IKE port")
 	cmd.Flags().BoolVar(&disableNat, "disable-nat", false, "Disable NAT for IPsec")
+	cmd.Flags().BoolVar(&allocGNet, "allocate-globalnet", true, "Allocate GlobalNet CIDR")
```
Comment (nyechiel) [inline pkg/subctl/cmd/join.go:93]:
```suggestion
	cmd.Flags().BoolVar(&allocGNet, "allocate-globalnet", true, "Allocate Globalnet CIDR")
```

**Comment by nyechiel** [comment]
Thanks for working on this, @renan-campos! 

@mangelajo @sridhargaddam IIRC a further discussion is needed to determine if we can support this, and especially if/how we can test this. Adding to our 0.8.1 board so that we can properly review this.

**Comment by nyechiel** [comment]
bump

**Review by mangelajo — APPROVED** [review]

**Comment by nyechiel** [comment]
TO DO: document the new flag

**Comment by nyechiel** [comment]
@mangelajo @tpantelis based on our discussion on this one, we can go ahead and merge it.

**Review by tpantelis — APPROVED** [review]

**Comment by nyechiel** [comment]
@tpantelis can you handle the docs part, please?

**Pull request merged**